server/route.go

```diff
@@ -1345,7 +1343,7 @@ func (c *client) processRemoteSub(argo []byte, hasOrigin bool) (err error) {
 		accPos = len(sub.origin) + 1
 	}
 	c.mu.Lock()
-	accountName := string(c.route.accName)
+	accountName := bytesToString(c.route.accName)
```

Comment:
Again, accountName used to lookup account, which may use this value to store as a key in map. We even possibly call `LookupOrRegisterAccount()` which would create a new account and user accountName for the name of the account.

Reply:
Can we change accName for routes?

Reply:
Not sure if you ask can we change it to be a string instead of []byte, or can the c.route.accName change for a given c.route. For the later, no. A route is either for a given account or not. The reason it is a []byte and not a string was because for processing of route messages, it was better to have it as a []byte to avoid conversions from string to []byte. That being said, if we were to make it a string, then we could change the places where we need it to be a []byte by using stringToBytes(). BUT, I think processing route messages is more frequent than the places where you modified to use bytesToString(), so I guess it is still better to keep it as a []byte.